aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -162,6 +165,7 @@ impl VMChangeSet {
             aggregator_v1_write_set,
             aggregator_v1_delta_set,
             delayed_field_change_set,
+            reads_needing_delayed_field_exchange,
```

Comment:
Is the read set in VMChangeSet guaranteed to be identical to captured reads? Can we still use capture reads in executor.rs when we need to access read set? Or will it result in an inconsistency?

Reply:
this reads are filtered down - to only those needing exchanges.

but for those - it will be the same